### PR TITLE
[FEATURE] account_info command

### PR DESCRIPTION
**Pull request by stevenzeiler**

### CODE REVIEW

**Comment by wilsonianb** [comment]
Update README

**Comment by stevenzeiler** [comment]
+r

File: README.md [hunk 1]
```diff
@@ -133,3 +134,24 @@ ripple-cli server_info --server wss://r.ripple.com:51233 | json
 }
 ````
 
+### Retrieving Server Info
```
Comment (wilsonianb) [inline README.md]: `Retrieving Account Info`

**Comment by wilsonianb** [comment]
LGTM

**Pull request merged**